MCMC/simulation.py

```diff
@@ -17,6 +33,7 @@ def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, wr
         self.accepted_moves = 0
         self.tps = None
         self.write_freq = write_freq
+        self.energy_func = energy_func
 
```

Comment:
It doesn't look like an attribute is being set for maximum allowed transition?

Reply:
A few lines above this line.